components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,30 @@
+import { List } from "./list";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function ListComponent({ component, elRef, eventHandlers }) {
+  const { classList, display, width, backgroundColor, type, listItems, color, fontSize, padding } = component;
+
+  const items = listItems || [];
+
+  const onClickHandler = (e, itemContent, itemChildren) => {
+    e.stopPropagation();
+    eventHandlers.onItemClick({ itemContent, itemChildren });
+  };
+
+  const options = {
+    type: type === "ol" ? "ol" : "ul",
+    style: { color, fontSize, padding },
```

Comment:
what do we apply this ' style` to? for List or ListItem?

Reply:
Apply to the ListItem